src/components/translation-builder/translation-builder.module.scss

```diff
@@ -0,0 +1,96 @@
+@use '@carbon/layout';
+@use '@openmrs/esm-styleguide/src/vars' as *;
+
+.translationBuilderContainer {
+  overflow-y: scroll;
+  overflow-x: hidden;
+  height: 95vh;
+}
+
+.title {
+  font-size: 1.25rem;
+  margin-bottom: layout.$spacing-05;
+}
+
+.languageSelector {
+  .languageSelector {
+    white-space: nowrap;
+    display: inline-block;
+  }
+}
+
+.translationEditor {
+  margin-top: layout.$spacing-05;
+}
+
+.translationRow {
+  display: grid;
+  grid-template-columns: 1fr 1fr auto;
+  margin-bottom: layout.$spacing-05;
+  padding: layout.$spacing-03 0;
+  border-bottom: 1px solid #e0e0e0;
```

Comment:
For colors, please use the colors from the styleguide, which in this case would be - 
https://github.com/openmrs/openmrs-esm-core/blob/75293696602b6aa61341bd50357b8cda9f33086f/packages/framework/esm-styleguide/src/_vars.scss#L5